src/main/java/frc/robot/subsystems/hood/Hood.java

```diff
@@ -55,7 +57,7 @@ public MutableMeasure<Angle> getAngle() {
     }
 
     public boolean atSetpoint() {
-        return io.atSetpoint();
+        return Utils.epsilonEquals(inputs.powerSetpoint * 12, inputs.voltage.in(Units.Volts));
```

Comment:
when you use percent output you use open-loop control, there is no setpoint. This method should return whether the angle setpoint is reached when using closed-loop control.

Reply:
52fd9db

Reply:
012b58c